zulip_bots/zulip_bots/bots/salesforce/doc.md

```diff
@@ -0,0 +1,52 @@
+# Salesforce bot
+
+The Salesforce bot can get records from your Salesforce database.
+It can also show details about any Saleforce links that you post.
+
+## Setup
+
+1. Create a user in Saleforce that the bot can use to access Salesforce.
+Make sure it has the appropriate permissions to access records.
+2. In `salesforce.conf` paste the Saleforce `username`, `password` and
+`security_token`.
+3. Run the bot as explained [here](https://zulipchat.com/api/running-bots#running-a-bot)
+
+## Configuration (Advanced)
```

Comment:
If this is not required, the title should be smt. like "Optional Configuration (Advanced)"